src/DynamicFormMultiSelect.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div>
+    <div :aria-label="formControl.label ? formControl.label : groupLabel">
         <tree-select :multiple="true"
                      :clearable="false"
```

Comment:
```suggestion
                     :clearable="false"
                     :aria-label="formControl.label ? formControl.label : groupLabel"
```

I think the right place for aria-label is within treeSelect.